src/PathFilter.cpp

```diff
@@ -0,0 +1,91 @@
+#include "../includes/PathFilter.h"
+
+#pragma unmanaged
+
+static const std::string doubleStarPrefix("**/");
+
+static bool prefix(const std::string &prefix, const std::string &str) {
```

Comment:
```suggestion
static bool startsWith(const std::string &prefix, const std::string &str) {
```

I believe `startsWith` would be clearer.